### PR TITLE
BAH-1792 | Added validation(file size and extension) for photo upload

### PR DESCRIPTION
**Pull request by SooryaKumaranC-tw**
Co-authored-by: deepthi-m <deepthi.mantena@thoughtworks.com>

### CODE REVIEW

File: ui/app/common/photo-capture/directives/capturePhoto.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ angular.module('bahmni.common.photoCapture')
             };
 
             scope.uploadImage = function () {
-                if (this.files && this.files[0]) {
+                if (this.files && this.files[0] && this.files[0].size <= 500000) {
```
Comment (N0-man) [inline ui/app/common/photo-capture/directives/capturePhoto.js:133]: Wouldn't it be better to externalize the size as configuration?

Comment (SooryaKumaranC-tw) [reply ui/app/common/photo-capture/directives/capturePhoto.js:133]: Sure, we will discuss and update accordingly.

**Comment by gsluthra** [comment]
@SooryaKumaranC-tw : Please incorporate @N0-man comments.

File: ui/app/common/photo-capture/directives/capturePhoto.js (hunk 1 above)
Comment (SooryaKumaranC-tw) [reply ui/app/common/photo-capture/directives/capturePhoto.js]: As we discussed we have externalized the size(**imageUploadSize**) as configuration. Also to not choke up the server capacity we have added maximum size(**maxImageUploadSize**) for the upload.

File: ui/app/common/photo-capture/directives/capturePhoto.js [hunk 2]
```diff
@@ -130,7 +134,18 @@ angular.module('bahmni.common.photoCapture')
             };
 
             scope.uploadImage = function () {
-                if (this.files && this.files[0]) {
+                if (this.files && this.files[0] && this.files[0].type) {
+                    var fileType = this.files[0].type;
+                    if (!fileType.startsWith('image/')) {
+                        uploadConfirmImageButton.prop('disabled', true);
+                        alert('File uploaded must be an image');
```
Comment (gsluthra) [inline ui/app/common/photo-capture/directives/capturePhoto.js:141]: These are user messages, and should not be hardcoded in english. Need to be translated.

Comment (SooryaKumaranC-tw) [reply ui/app/common/photo-capture/directives/capturePhoto.js:141]: Sure Gurpreet. This current PR is a duplicate one. I will complete the above comment in a separate card. So will create a separate card and will raise PR accordingly. So closing this PR.

**Comment by SooryaKumaranC-tw** [comment]
Duplicate PR([BAH-1792-photo-upload](https://github.com/Bahmni/openmrs-module-bahmniapps/tree/BAH-1792-photo-upload)). It is already merged.

**Pull request closed without merging**